cli/marketplace/build.py

```diff
@@ -38,7 +38,6 @@
 ASSETS = {
     "example": ("example", "src/{}"),
     "source": ("spec.filename", "src/{}"),
-    "function": "src/function.yaml",
```

Comment:
I saw that we have a test in mlrun named `test_hub_get_asset_from_default_source` that verify that these assets exist in the catalog file. we need to make sure that we also update the mlrun test as well. 
https://github.com/mlrun/mlrun/blob/development/server/py/services/api/tests/unit/api/hub/test_hub.py#L318